internal/exec/vendor_utils.go

```diff
@@ -19,6 +19,9 @@ import (
 	u "github.com/cloudposse/atmos/pkg/utils"
 )
 
+// Dedicated logger for stderr to keep stdout clean of detailed messaging, e.g. for files vendoring.
+var StderrLogger = log.New(os.Stderr)
```

Comment:
We do not need to introduce a new logger. We use `log.*` everywhere now without this.